### PR TITLE
SECURITY.md

### PR DESCRIPTION
**Pull request by sepandhaghighi**
#### What does this implement/fix? Explain your changes.
- `SECURITY.md` added

### CODE REVIEW

**Comment by sepandhaghighi** [comment]
@AHReccese GitHub has changed the 'macOS-latest' alias to 'arm64' architecture which doesn't contain some Python versions. Consequently, the 'macOS' version has been fixed to a stable version (13).

**Comment by AHReccese** [comment]
> @AHReccese GitHub has changed the `macOS-latest`alias to `arm64` architecture which doesn't contain some Python versions. Consequently, the `macOS` version has been fixed to a stable one (13).

Impressive, well done.

**Comment by AHReccese** [comment]
I thought it was a simple cache miss, which I ran into before and by re-running it gets fixed(which it doesn’t here).
@sepandhaghighi

File: SECURITY.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
+# Security policy
+
+## Supported versions
+
+| Version       | Supported          |
```
[the remark this answers is not in the document]
Comment (sepandhaghighi) [reply SECURITY.md:5]: This will be updated after the official release of that version.

**Pull request merged**